.github/workflows/build-nix.yml

```diff
@@ -1,22 +1,25 @@
-name: nix-build
-on:
-  pull_request:
-  push:
-    branches: [main, 'release-v**']
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          # Nix Flakes doesn't work on shallow clones
-          fetch-depth: 0
-      - uses: cachix/install-nix-action@v17
-        with:
-          install_url: https://releases.nixos.org/nix/nix-2.10.3/install
-      - uses: cachix/cachix-action@v10
-        with:
-          name: centrifuge-chain
-          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
-      - name: Build centrifuge-chain
-        run: nix build --no-allow-dirty -L
\ No newline at end of file
+#### NOTE: This job is disabled until nix supports the latest cargo workspace features.
+#### See: https://github.com/NixOS/nixpkgs/pull/217232
```

Comment:
Why do we depend on those new features? Substrate?

Reply:
See @wischli's comment on that thread: https://github.com/NixOS/nixpkgs/pull/217232#issuecomment-1491319021